custom_components/teslafi/sensor.py

```diff
@@ -246,6 +246,26 @@
         available=lambda u, d, h: u and d.tpms.rear_right,
     ),
     # endregion
+    # region TeslaFi API Counts
+        TeslaFiSensorEntityDescription(
+        key="commands",
+        name="API Commands",
+        icon="mdi:counter",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=True,
+        value=lambda d, h: d.api_requests.commands,
```

Comment:
Pretty sure we don't need this 🤔 The parent class will take care of returning essentially `d[key]`, which will already work as it is because you update vehicle data with the request counts data already.